src/js/jenkins_loader.js

```diff
@@ -0,0 +1,97 @@
+ 'use strict;'
```

Comment:
Just opened a PR to squash a few of these gremlin semicolons wanting in on the quote action! ;)

Reply:
I prefer to write everything in JSX and allow the transpiler to deal with them :)